app/src/main/java/com/iic/shopingo/services/FacebookConnector.java

```diff
@@ -0,0 +1,70 @@
+package com.iic.shopingo.services;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import bolts.Task;
+import com.facebook.FacebookRequestError;
+import com.facebook.Request;
+import com.facebook.Response;
+import com.facebook.Session;
+import com.facebook.model.GraphLocation;
+import com.facebook.model.GraphPlace;
+import com.facebook.model.GraphUser;
+import com.iic.shopingo.dal.models.UserInfo;
+
+/**
+ * Created by ifeins on 3/3/15.
+ */
+public class FacebookConnector {
+  public static Task<UserInfo> connectWithFacebook(Session session) {
```

Comment:
If it's already in a `FacebookConnector` class it might as well be named to simply `login` to match the `logout` method below.